docs/releases/4.2.4.txt

```diff
@@ -12,3 +12,6 @@ Bugfixes
 * Fixed a regression in Django 4.2 that caused a crash of
   ``QuerySet.aggregate()`` with aggregates referencing window functions
   (:ticket:`34717`).
+
+* Fixed a regression in Django 4.2 that caused a crash when grouping by a
+  ``F`` reference in a subquery (:ticket:`34748`).
```

Comment:
In the docs, I see a mix of `an F expression` and `a F object`. I would have thought that we should use `an F <whatever>`. What do you think?

Reply:
I'd chop ``` ``F`` ```.